### DISCUSSION TITLE
Missing files: MiKTeX210200-.....dll when generating PDF file

### DISCUSSION THREAD

**Discussion by JuliaSzymanska** [MiKTeX/miktex#770] [Q&A] [answered]
MikTex worked properly before reinstallation. After installing the programme I get an error whenever I try to generate a PDF, that objects MiKTeX210200-web2c.dll, MiKTeX210200-png.dll, MiKTeX210200-kpathsea.dll, MiKTeX210200-texmf.dll are missing. I have just uninstalled and installed the programme. I have not deleted any files or directories. 

Error in Polish language:
![image](https://user-images.githubusercontent.com/45401422/112829584-9b97b300-9091-11eb-85ac-1dadb650e2cc.png)

OS: Windows 10 Home, 20H2, 64-bit, 
MiKTeX version: 21.3
Updated: 29.03.2021
Root privileges: yes
Admin mode: yes

**Comment by edocevoli** [comment, marked as answer]
Seems  that executables like `pdflatex.exe`  are not up-to-date. You can create new executables as follows:

```sh
initexmf --mklinks --force
```

**Reply by JuliaSzymanska** [reply]
It works, thank you!

**Reply by bitepen** [reply]
Works fine, thank you sir!

**Comment by Euniceisnice** [comment]
It does not work for me. Could you provide more details?
![Screenshot 2022-01-21 233333](https://user-images.githubusercontent.com/43316717/150624733-ff6c457d-64c2-463e-8ddc-7c51a28727cb.png)

![Screenshot 2022-01-21 233505](https://user-images.githubusercontent.com/43316717/150624763-feeeff62-bc23-45b0-ac95-f865aabf0225.png)
